setup.cfg

```diff
@@ -34,9 +28,7 @@ classifiers =
 
 [options]
 packages = asttokens
-install_requires = 
-    six >= 1.12.0
-    typing; python_version < "3.5"
+install_requires = six >= 1.12.0
```

Comment:
Doesn't this PR actually make `six` irrelevant and unused ?
```suggestion
```

Reply:
six usage remains in places

Reply:
I see. I expect applying https://github.com/gristlabs/asttokens/pull/117#issuecomment-1999541944 would solve that, but it could also be done as a follow up patch.

Reply:
Please have look on https://github.com/gristlabs/asttokens/pull/117/commits/de0a2e7307998fea712353fa0865ad0a1b4fec0e where is patch which removes use of six from all places.